applications/connecting_applications_to_services/understanding-service-binding-operator.adoc

```diff
@@ -0,0 +1,83 @@
+[id="understanding-service-binding-operator"]
+= Understanding Service Binding Operator
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: understanding-service-binding-operator
+
+toc::[]
+
+:FeatureName: Service Binding Operator
+
+Application developers need access to backing services to build and connect workloads. Connecting workloads to backing services is always a challenge as each service provider suggests a different way to access their secrets and consume them in a workload. In addition, manual configuration and maintenance of this binding together of workloads and backing services makes the process tedious, inefficient, and error-prone.
```

Comment:
Is "backing services" a common term in OpenShift docs?

Reply:
as adverb
Do not use to mean "because" or "while". (IBM SG)

Reply:
makes:

Incorrect verb form with compound subject
It appears that the singular verb *makes* does not agree with the plural compound subject *manual configuration and maintenance of this binding together of workloads and backing services*. Consider changing the verb to the plural form.

A compound subject includes two or more nouns (or pronouns, noun clauses, or noun phrases) and may be singular or plural. When the nouns are joined with the word and (an apple and a banana), the compound subject is usually plural, even if the individual nouns are singular. A plural subject requires a plural verb. On occasion, a compound subject uses two nouns to describe one singular subject (spaghetti and meatballs, my sister and best friend).

(Grammarly)